core/modules/node/node.theme.inc

```diff
@@ -32,6 +32,22 @@ function theme_node_preview_banner_form($variables) {
   return $output;
 }
 
+/**
+ * Returns HTML for node_preview().
```

Comment:
Would not *Returns HTML for node previews.* be preferable?

Reply:
FTR, the description in the respective function in D7 has the following:
> Returns HTML for a node preview for display during node creation and editing.

See https://api.drupal.org/api/drupal/modules%21node%21node.pages.inc/function/theme_node_preview/7.x

Reply:
Went with @klonos suggestion